doc/conf.py

```diff
@@ -95,6 +97,18 @@ def _check_deps():
   'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
   }
 
+explicit_order_folders = ['../examples/pyplots',
+                          '../examples/statistics',
+                          '../examples/color',
+                          '../examples/pylab_examples',
+                          '../examples/showcase',
+                          '../tutorials/introductory',
```

Comment:
Where should I be looking to find this? In the gallery page, I don't think I see any tutorials after showcase?